frontend/src/pages/modelRegistry/screens/RegisterModel/RegisterModel.tsx

```diff
@@ -75,7 +84,17 @@ const RegisterModel: React.FC = () => {
                     name="model-name"
                     value={modelName}
                     onChange={(_e, value) => setData('modelName', value)}
+                    validated={isModelNameValid ? 'default' : 'error'}
                   />
+                  {!isModelNameValid && (
+                    <FormHelperText>
+                      <HelperText>
+                        <HelperTextItem variant="error">
+                          Cannot exceed 128 characters
```

Comment:
can we use the `NAME_CHARACTER_LIMIT` const here too?

Reply:
also can we rename `NAME_CHARACTER_LIMIT` to `MR_CHARACTER_LIMIT` or anything that states this is solely for MR?